arbitrum-docs/stylus/concepts/stylus-gas.md

```diff
@@ -1,20 +1,29 @@
 ---
-title: 'Gas Costs in Stylus'
-sidebar_label: 'Gas Costs in Stylus'
-description: 'A conceptual overview of the pricing models of Stylus, including the relationshup between gas and ink.'
+title: 'Gas and ink (Stylus)'
+sidebar_label: 'Gas and ink'
+description: 'A conceptual overview of gas and ink, the primitives that Stylus uses to measure the cost of WASM activation, compute, memory, and storage.'
 author: rachel-bousfield
 sme: rachel-bousfield
 target_audience: 'Developers deploying smart contracts using Stylus.'
 sidebar_position: 3
 ---
 
-This conceptual overview is for Stylus Developers who want to understand the pricing models that Stylus uses to assess the price of various opcodes and host I/Os, along with its native unit of payment, ink. If interested in the latest ink and gas measurements of WASM opcodes and host I/Os, please visit the [Opcode and Host I/O pricing reference](https://docs.arbitrum.io/stylus/reference/opcode-hostio-pricing).
+**Gas and ink** are the pricing primitives that Stylus uses to determine how much it costs to handle specific opcodes and host I/Os. For an overview of specific opcode and host I/O costs, see [Opcode and host I/O pricing](/stylus/reference/opcode-hostio-pricing).
```

Comment:
```suggestion
**Ink** is a new pricing primitive that Stylus uses to determine how much it costs to handle specific opcodes and host I/Os, which is needed as many WASM opcodes cost less than `1 gas`. For an overview of specific opcode and host I/O costs, see [Opcode and host I/O pricing](/stylus/reference/opcode-hostio-pricing).
```

Reply:
Definitely open to continue chiseling this line

Reply:
I'd recommend keeping this as-is - the title of the document sets the expectation that this explains Gas and Ink, which it does.

We can distinguish between gas and ink, and elaborate on the rationale underlying the ink primitive, within the body of the page :)